src/Application/UI/Presenter.php

```diff
@@ -1047,7 +1047,7 @@ public static function argsToParams($class, $method, & $args, $supplemental = []
 				));
 			}
 
-			if ($args[$name] === $def || ($def === NULL && is_scalar($args[$name]) && (string) $args[$name] === '')) {
```

Comment:
Because as far as I can tell `is_scalar($args[$name]) && (string) $args[$name] === ''` is equivalent to `$args[$name] === '' || $args[$name] === false`.